File: CINEMATRIX.API.Contracts/IncomingOutgoing/HallDTO.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace CINEMATRIX.API.Contracts.IncomingOutgoing
{
    public class HallDTO
    {
        public string Name { get; set; }
        public string Description { get; set; }
        public string SoundSystem { get; set; }
        public string Projector { get; set; }
        public string Screen { get; set; }
        public string ScreenResolution { get; set; }
    }
}

File: CINEMATRIX.Client/src/app/core/models/hall/hall-dto.ts
import { SeatFoundIncomingDto } from "../seat/seat-found-incoming-dto"

export interface HallDto {
  name: string,
  description: string,
  soundSystem: string,
  projector: string,
  screen: string,
  screenResolution: string
  seats: SeatFoundIncomingDto[]
}
